api/src/pcapi/core/api_token/api_token.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from datetime import timedelta
+import enum
+
+from flask import current_app as app
+
+from pcapi.core.users.exceptions import InvalidToken
+from pcapi.core.users.utils import decode_jwt_token
+from pcapi.core.users.utils import encode_jwt_payload
+
+
+class TokenType(enum.Enum):
+    TEST_TOKEN_TYPE_1 = "test1"  # this type is used in the tests only
+    TEST_TOKEN_TYPE_2 = "test2"  # this type is used in the tests only
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Pour moi c'est KO
Pas de données de test hors des tests
Rien ne t'empèche d'étendre cet Enum ou de le mocker dans les tests